modules/ROOT/pages/dynamic-expression-field.adoc

```diff
@@ -87,7 +87,7 @@ You can combine your simple value with a DataWeave expression, if you know the D
 
 ====
 
-* Remove a data pill from a field by clicking in the field immediately after it and typing your Delete or Backspace key.
+* Remove a data element from a field by clicking in the field immediately after it and typing your Delete or Backspace key.
```

Comment:
"clicking" instead of "typing"

Reply:
One types a key on a keyboard and clicks on object onscreen in a UI.